nextcloud/android
# Open document tree

TODO
- [ ] same approach for public Cursor queryChildDocuments(String parentDocumentId, String[] projection, String sortOrder)

To test
- use at least Android O
- use https://github.com/android/storage/tree/master/ActionOpenDocumentTree
- open NC via lower left buttom
- select directory
- see directory listing in android test app
